src/main/java/ca/on/oicr/gsi/dimsum/util/reporting/reports/CaseTatReport.java

```diff
@@ -139,44 +129,74 @@ private static String getSortedProjectNameOrPipeline(Case kase,
         .collect(Collectors.joining(", "));
   }
 
-  private static List<CaseFilter> convertParametersToFilters(Map<String, String> parameters) {
+  private static List<CaseFilter> convertParametersToFilters(JsonNode parameters) {
     List<CaseFilter> filters = new ArrayList<>();
-    parameters.forEach((key, value) -> {
-      CaseFilterKey filterKey = CaseFilterKey.valueOf(key.toUpperCase());
-      CaseFilter filter = new CaseFilter(filterKey, value);
-      filters.add(filter);
+    parameters.fields().forEachRemaining(entry -> {
+      try {
+        CaseFilterKey filterKey = CaseFilterKey.valueOf(entry.getKey().toUpperCase());
+        String[] values = entry.getValue().asText().split(",");
+        for (String value : values) {
+          CaseFilter filter = new CaseFilter(filterKey, value.trim());
+          filters.add(filter);
+        }
+      } catch (IllegalArgumentException e) {
+        throw new BadRequestException(
+            "Invalid filter key: " + entry.getKey() + ". " + e.getMessage());
+      }
```

Comment:
There are static methods in `MvcUtils` that you can use for parsing the case filters